regparser/tree/xml_parser/reg_text.py

```diff
@@ -177,9 +178,21 @@ def build_from_section(reg_part, section_xml):
                 n = Node(node_text[0], [], [m], source_xml=ch)
                 n.tagged_text = unicode(node_text[1])
                 nodes.append(n)
+
             if node_text[0].endswith('* * *'):
                 nodes.append(Node(label=[mtypes.INLINE_STARS]))
 
+    # special handling for reg J, whose definitions are inexplicably
+    # unnumbered and yet required to belong to node that precedes them
+    if reg_part == '1010' and nodes != [] and nodes[-1].text.find('As used in this part:') > -1:
+        for i, [plain_text, tagged_text] in enumerate(section_texts):
+            subnode = Node(plain_text, label=['b', str(i)], title="")
+            subnode.tagged_text = tagged_text
+            nodes[-1].children.append(subnode)
```

Comment:
Thanks, I'll take a look. This was added in to get through this one section, and also because I don't entirely have the entire picture of the code in my head yet. Obviously having special exceptions is not great; it's a stopgap until I figure out a better solution.